playground/frontend/lib/modules/sdk/components/sdk_selector.dart

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter/material.dart';
+import 'package:playground/config/theme.dart';
+import 'package:playground/modules/sdk/models/sdk.dart';
+
+typedef void SetSdk(SDK sdk);
+
+class SDKSelector extends StatelessWidget {
+  final SDK sdk;
+  final SetSdk setSdk;
+
+  const SDKSelector({Key? key, required this.sdk, required this.setSdk})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.symmetric(
+        vertical: 0.0,
```

Comment:
This comment is not blocking of the PR.

In my experience as the flutter app gets larger, hardcoding values like this becomes problematic.  One idea is to place them in the constants file/folder as

```
const kVerticalPadding = ...
```

which isn't flexible but just as an example.  As the app grows, perhaps we'll see ways to refactor these configuration like areas to improve readability.

Reply:
Thank you, that will be definitely very helpful!!